test/step-definitions/linking-providers-outbound.step.ts

```diff
@@ -0,0 +1,157 @@
+ their names indented and be marked with a '-'. Email address can be added
+ optionally within square brackets <email>.
+
+  - Kevin Leyow <kevin.leyow@modusbox.com>
+
+ --------------
+ ******/
+import { Server, ServerInjectResponse } from '@hapi/hapi'
+import { ServerAPI, ServerConfig } from '~/server'
+import { defineFeature, loadFeature } from 'jest-cucumber'
+import Config from '~/shared/config'
+import Handlers from '~/handlers'
+import index from '~/index'
+import path from 'path'
+import { NotificationCallback, Message, PubSub } from '~/shared/pub-sub'
+import { RedisConnectionConfig } from '~/shared/redis-connection'
+import { thirdparty as tpAPI } from '@mojaloop/api-snippets'
+
+const apiPath = path.resolve(__dirname, '../../src/interface/api-outbound.yaml')
+const featurePath = path.resolve(__dirname, '../features/linking-providers-outbound.feature')
+const feature = loadFeature(featurePath)
+
+jest.mock('@mojaloop/sdk-standard-components', () => {
+  return {
+  MojaloopRequests: jest.fn(),
+    ThirdpartyRequests: jest.fn(() => ({
+      getServices: jest.fn(() => Promise.resolve())
+    })),
+    WSO2Auth: jest.fn(),
+    Logger: {
+      Logger: jest.fn(() => {
+        const methods = {
+          // log methods
+          log: jest.fn(),
+          configure: jest.fn(),
+          // generated methods from default levels
+          verbose: jest.fn(),
+          debug: jest.fn(),
+          warn: jest.fn(),
+          error: jest.fn(),
+          trace: jest.fn(),
+          info: jest.fn(),
+          fatal: jest.fn()
+        }
+        return {
+          ...methods,
+          push: jest.fn(() => methods)
+        }
+      })
+    },
+    request: jest.fn()
+  }
+})
+
+jest.mock('~/shared/pub-sub', () => {
+  let handler: NotificationCallback
+  let subId = 0
+  return {
+    PubSub: jest.fn(() => ({
+      isConnected: true,
+      subscribe: jest.fn(
+        (_channel: string, cb: NotificationCallback) => {
+          handler = cb
+          return ++subId
+        }
+      ),
+      unsubscribe: jest.fn(),
+      publish: jest.fn(
+        async (channel: string, message: Message) => {
+          return handler(channel, message, subId)
+        }
+      ),
+      connect: jest.fn(() => Promise.resolve()),
+      disconnect: jest.fn()
+    }))
+  }
+})
+
+async function prepareOutboundAPIServer (): Promise<Server> {
+  const serverConfig: ServerConfig = {
+    port: Config.OUTBOUND.PORT,
+    host: Config.OUTBOUND.HOST,
+    api: ServerAPI.outbound
+  }
+  const serverHandlers = {
+    ...Handlers.Shared,
+    ...Handlers.Outbound
+  }
+  return index.server.setupAndStart(serverConfig, apiPath, serverHandlers)
+}
+
+defineFeature(feature, (test): void => {
+  let server: Server
+  let response: ServerInjectResponse
+
+  afterEach((done): void => {
+    jest.resetAllMocks()
+    jest.resetModules()
+    server.events.on('stop', done)
+    server.stop()
```

Comment:
let stop server ASAP so the unit tests will not hung-up
```suggestion
    server.stop({ timeout:0 })
```

Reply:
done